packages/cxl-ui/src/components/cxl-tabs-slider.js

```diff
@@ -12,7 +12,13 @@ export class CXLTabsSliderElement extends Tabs {
   get _scrollOffset() {
     const theme = this.getAttribute('theme').split(' ');
 
-    if (theme.includes('cxl-category-accordion') || theme.includes('cxl-slider-dashboard-header')) {
+    const themes = [
```

Comment:
I don't get it. Is this supposed to fix the scrolling snap behavior? That is not what this function does. Refer to my comment [here](https://app.clickup.com/t/86ayn4zg9?comment=90140012778694&threadedComment=90140012833648)

Reply:
@freudFlintstone Yes. Those scroll snap styles were not working because this slider already had the cxl-course-slider theme attached. Before my fix.

Reply:
OK, but isn't it confusing to include several themes with the component being used in a different context? Are you sure the styles are not conflicting? I feel it would be easier to just use the boolean attribute to turn on the feature, specifically.

Reply:
Yes, it is confusing, but there was a similar solution there before, where the theme did not belong to that context either. I just made the code more compact and clear. Without this part, scrolling for one card does not work anywhere.